### PR TITLE
Add icon set

### PR DESCRIPTION
**Pull request by manu29d**
[feather](https://github.com/feathericons/feather#feather)
[https://feathericons.com/](https://feathericons.com/)
[React-feather](https://github.com/carmelopullara/react-feather)
screenshot 2018-12-26 at 8 24 59 pm

### CODE REVIEW

**Comment by manu29d** [comment]
@sushantahujaa Please review

**Comment by manu29d** [comment]
@sushantahujaa Please review.

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
+    "react-feather": "^1.1.5",
```
Comment (sushantahujaa) [inline package.json:8]: what is this for? ` testing slack integration `

Comment (manu29d) [reply package.json:8]: Slack integration works

Comment (manu29d) [reply package.json:8]: To answer your question, check links on the PR description.

**Review by mounika-puppala — APPROVED** [review]

**Pull request merged**